metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -158,6 +158,17 @@ object ServerCommands {
        |""".stripMargin
   )
 
+  val RunScalafix = new ParametrizedCommand[TextDocumentPositionParams](
+    "scalafix-run",
+    "Run all Scalafix Rules",
+    """|Run all the supported scalafix rules in your codebase.
+       |
+       |If the rules are missing please add them to user configuration `metals.scalafixRulesDependencies`.
```

Comment:
this is in coursier format, right?

Reply:
That's what I am thinking. It would be better to get that info from sbt, but it's not currently exposed. I am going to add a bunch of docs for that